app/modules/users/permissions/rules.py

```diff
@@ -77,9 +77,11 @@
         'is_admin',
         'is_researcher',
     ],
-    ('Individual', AccessOperation.READ): ['is_researcher'],
+    ('Individual', AccessOperation.READ): ['is_staff', 'is_data_manager', 'is_researcher'],
```

Comment:
the is_staff should not be necessary for any of these as that should be caught by the owner_or_priviliged check

Reply:
Mark was finding otherwise in his tests making this ticket, said a staff user in particular could not read all these objects if they were owned. I say it's better to have possible redundancy that definitely works than non-redundancy that's questionable.